exchanges/gateio/gateio_websocket_option.go

```diff
@@ -591,7 +591,7 @@ func (g *Gateio) processOptionsOrderbookSnapshotPushData(event string, incoming
 			Exchange:        g.Name,
 			Pair:            currencyPair,
 			LastUpdated:     updatePushedAt,
-			UpdatePushedAt:  updatePushedAt,
+			LastPushed:      updatePushedAt,
```

Comment:
Since you are renaming all these, could you rename the `updatePushedAt` variable in here and in orderbook_types.go >> `option struct` and other variables of this kind to `lastPushed` just for consistency.

Just a suggestion.

Reply:
[d110db4](https://github.com/thrasher-corp/gocryptotrader/pull/1917/commits/d110db43e6143b4df2a5917ec727c39b7f82b29a)